### ISSUE TITLE
RPC ```eth_getBalance``` returns 0 SHM in Metamask

### ISSUE THREAD

**Issue by AndreiW3b3** [Shardeum/shardeum-bug-reporting#44]
##### What is the issue?

Metamask shows 0 SHM for an account that has more than 0 SHM on the Shardeum explorer. 
    
#### What impact does the issue have?

Cannot send SHM between wallets. 

##### How to reproduce the issue? 

Use Metamask with Betanet 1.X with SHM in the wallet address, then wait for the balances to randomly go to 0 SHM.
   
##### What other resources can you share regarding this issue?

RPC method ```eth_getBalance```: 

https://www.quicknode.com/docs/ethereum/eth_getBalance

**Comment by AndreiW3b3** [comment]
Thank you for the fast response. So the announcement as well. Keep it up 🔥Sent from my iPhoneOn 8 Feb 2023, at 01:16, Marcus Wentz ***@***.***> wrote:﻿
This is due to an RPC issue we are looking into.

**Comment by MarcusWentz** [comment]
This is working now in a Betanet 1.1 test environment.

**Comment by MarcusWentz** [comment]
Betanet 1.1 appears to have RPC and Validator issues causing this. We are looking into this.

**Comment by AndreiW3b3** [comment]
Just after i instaled the last update i just noticed thae the ballance in my metamask started decreasing again

**Comment by MarcusWentz** [comment]
Q1. Does the balance go to 0 SHM randomly in Metamask?

Q2. Does the actual SHM balance decrease on the Shardeum Explorer?

**Comment by AndreiW3b3** [comment]
Q1. It does go to zero from time to time

Q2. It is decreasing also in my MetaMask. I was 4.90 and now I have 4.72

**Comment by MarcusWentz** [comment]
Q1. Did you stake with the address with the decreasing SHM balance? 

Q2. What is your wallet address with the decreasing Metamask balance?

**Comment by AndreiW3b3** [comment]
Yes I staked with the wallet the the balance is decreasing as I did not notice that before staking0x7eb1DD4eC345dF607885597EC9f8aEFb4EeEEd4aSent from my iPhoneOn 26 Mar 2023, at 04:23, Marcus Wentz ***@***.***> wrote:﻿
Q1. Did you stake with the address with the decreasing SHM balance?
Q2. What is your wallet address with the decreasing Metamask balance?

**Comment by swarna1101** [comment]
Extensive testing conducted on Metamask with Betanet 1.X revealed that the balances intermittently showed 0 SHM, despite having holdings on the Shardeum explorer. The issue has been rectified in the latest version, ensuring accurate and synchronized SHM balances between Metamask and the Shardeum explorer.
